core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/documents/UINewDocumentForm.java

```diff
@@ -83,10 +86,11 @@
    * Constructor.
    *
    */
-  public UINewDocumentForm() {
+  public UINewDocumentForm() throws Exception{
     this.documentService = this.getApplicationComponent(DocumentService.class);
     // Title textbox
     UIFormStringInput titleTextBox = new UIFormStringInput(FIELD_TITLE_TEXT_BOX, FIELD_TITLE_TEXT_BOX, null);
+    titleTextBox.addValidator(MandatoryValidator.class).addValidator(XSSValidator.class);
```

Comment:
I think the mandatory validator is not necessary
Before your fix, if you leave the title empty, there is already a message. So please remove this validator (dont forget to remove the import)